backend/Queries/middlewear.js

```diff
@@ -0,0 +1,108 @@
+let school = require("../db/db")
+
+
+const getAllClasses = (req,res)=>{
+    res.json({
+        status: "sucess",
+        classes: school.classes,
+        timestamp: Date(Date.now()).toString()
+    })
+}
+const addNewClass = (req,res) =>{
+    let newClassName = req.body["name"]
```

Comment:
Theres no need to key into it via bracket notation. `req.body.name` or `req.body.teacher` is enough. But thats all up to you